unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs

```diff
@@ -237,6 +237,12 @@ internal void OnUserProfileUpdated(UserProfile userProfile)
                 lastDisablePEXSentToKernel = pausedPEXScenesIds;
                 WebInterface.SetDisabledPortableExperiences(pausedPEXScenesIds.ToArray());
             }
+
+            List<ExperienceRowComponentView> loadedExperiences = view.GetAllAvailableExperiences();
+            foreach (ExperienceRowComponentView experience in loadedExperiences)
```

Comment:
Nitpicking here but let's use a `for`.

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2174/commits/a61743addd844e68fb665f47f81738851ec580dd. Thanks!